TombEngine/Game/control/los.cpp

```diff
@@ -136,38 +136,32 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 						if ((Objects[item->ObjectNumber].explodableMeshbits & ShatterItem.bit) &&
 							LaserSight)
 						{
-							//if (!Objects[item->objectNumber].intelligent)
-							//{
+							if (item->ObjectNumber == ID_GUARD_LASER)
```

Comment:
Same as with Tokyo's puna branch, I'd prefer to move such slot-specific hack out of `GetTargetOnLOS`.

Reply:
removed